controllers/drplacementcontrol.go

```diff
@@ -535,20 +535,30 @@ func (d *DRPCInstance) checkMetroFailoverPrerequisites(curHomeCluster string) (b
 func (d *DRPCInstance) checkRegionalFailoverPrerequisites() bool {
 	d.setProgression(rmn.ProgressionWaitForStorageMaintenanceActivation)
 
+	drRearrangedClusters := d.drClusters
+
+	var failoverCluster rmn.DRCluster
+
+	// find the failover cluster and move it to the first position in clusters array
```

Comment:
This comment explains what the code does which is very rarely needed. It is also wrong since the code does not move the failover cluster (it append it to the end) and the clusters is not an array (its a slice).